### PR TITLE
OPSEXP-2117 acs-deployment to rely on chart alfresco-connector-msteams

### PR DESCRIPTION
**Pull request by slohe1**
OPSEXP-2117

### CODE REVIEW

**Review by alxgomz — COMMENTED** [review]
There is no meaningful test here.
We should try that branch on develop branch of the integration pipeline given it's the only one that can really deploy and configure properly the instances.
Doing that require some communication that we will break develop

File: helm/alfresco-content-services/values.yaml [hunk 1]
```diff
@@ -160,60 +160,11 @@ ooiService:
...
-# -- Enable/Disable Alfresco Content Connector for Microsoft Teams
-msTeams:
+alfresco-connector-msteams:
+  # -- Enable/Disable Alfresco Content Connector for Microsoft Teams
   enabled: false
-msTeamsService:
-  replicaCount: 2
-  nodeSelector: {}
   image:
-    repository: quay.io/alfresco/alfresco-ms-teams-service
```
Comment (gionn) [inline helm/alfresco-content-services/values.yaml:170]: keep it just for reference together with tag

**Review by gionn — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/7.0.N_values.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ alfresco-sync-service:
 ooiService:
   image:
     tag: 1.1.0.1
-msTeams:
+alfresco-connector-msteams:
```
Comment (gionn) [inline helm/alfresco-content-services/7.0.N_values.yaml:59]: document this change in the upgrade md

**Review by gionn — CHANGES_REQUESTED** [review]

File: docs/helm/upgrades.md [hunk 3]
```diff
@@ -9,6 +9,22 @@ Here follows a more detailed explanation of any breaking change grouped by versi
...
+repository.
+
+Accordingly to this chart rename, also the related values has been moved from
+`.Values.msTeamsService` to
+`.Values.alfresco-connector-msteams`.
+
+Additionally values in the 7.0.N_values.yaml has been moved from
+`.Values.msTeams` to
+`.Values.alfresco-connector-msteams`.
+
```
Comment (gionn) [inline docs/helm/upgrades.md:27]:
```suggestion
```

don't see the need of saying it again, the previous paragraph should be enough

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**